src/actions/admin_actions.js

```diff
@@ -0,0 +1,39 @@
+import axios from 'axios';
+import { SUSPEND_USER, UNSUSPEND_USER, DELETE_USER } from './types';
+
+export const suspendUser = ({ id, suspended }) => async dispatch => {
+  const request = await axios({
+    method: 'post',
+    url: 'http://localhost:9000/api/v1/users/suspend',
```

Comment:
@danielnmai updated frontend with env... so instead of `localhost...` use `REACT_APP_API_URL_DEV`

Reply:
Will help with the change.